README.md

```diff
@@ -788,6 +790,38 @@ endpoints:
 ```
 
 
+#### Configuring Pushover alerts
+| Parameter                                       | Description                                                                                | Default                      |
+|:------------------------------------------------|:-------------------------------------------------------------------------------------------|:-----------------------------|
+| `alerting.pushover`                             | Configuration for alerts of type `pagerduty`                                               | `{}`                         |
+| `alerting.pushover.application-key`             | Pushover Application key                                                                   | `""`                         |
+| `alerting.pushover.user-key`                    | User or group key                                                                          | `""`                         |
```

Comment:
Are you sure `application-key` and `user-key` are the terms we want to use?

I've never used pushover before, but would it not be more accurate to call it `application-key` `token` and `user-key` `user` just for the sake of being consistent with the payload?

That'd future-proof it too, in case one day we add support for [groups](https://pushover.net/api/groups)

Reply:
Pushover calls the API keys either keys or tokens. I would rather use `application-token` and `user-key` so that users know exactly what they have to put in. `token` would be too generic for me.

Reply:
I'm fine with that.
Can you update `application-key` to `application-token`? 🙏

Reply:
Done